package.json

```diff
@@ -57,6 +60,9 @@
     "url-pattern": "1.0.3",
     "wait-on": "7.0.1"
   },
+  "resolutions": {
+    "@types/react": "^17.0.62"
+  },
```

Comment:
Note: this is needed to fix [this issue](https://github.com/styled-components/styled-components/issues/3721) of `@types/styled-components`